awx/main/models/ha.py

```diff
@@ -131,6 +131,13 @@ def is_lost(self, ref_time=None):
         grace_period = 120
         return self.modified < ref_time - timedelta(seconds=grace_period)
 
+    def mark_offline(self, on_good_terms=False):
+        self.cpu = self.cpu_capacity = self.memory = self.mem_capacity = self.capacity = 0
```

Comment:
should we also mark the instance as disabled (`self.enabled = False` iirc)?

Reply:
No, the `enabled` field should be totally separate. It may actually come-and-go while being disabled, because the enabled flag is something the user will have control over. These fields should become the sole domain of the health checks.